pages/glossary/background-colors-of-element.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Background Colors Of Element
+key: background-colors-of-element
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+The colors of all pixels, excluding the [foreground colors](#foreground-colors-of-text) pixels, in the element's [bounding client rectangle](https://drafts.csswg.org/cssom-view/#dom-element-getboundingclientrect).
```

Comment:
Foreground color only applies to text nodes, not to elements. Also from the look if it you're including things like img elements, other elements that are rendered on top or underneath it, and scrollbars in the background. Either this shouldn't be called background, or we need a stronger definition.

Reply:
I am including everything in the background. And I don't see why that is a problem. If one element has an image for the background and the other doesn't then they have distinguishing background colours. 

This isn't different from what was already the definition of background colors of text.